packages/opentelemetry-propagation-jaeger/package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "@opentelemetry/propagation-jaeger",
+  "version": "0.3.2",
+  "description": "OpenTelemetry propagation Jaeger provide HTTP header propagation for systems that using Jaeger HTTP header format",
+  "main": "build/src/index.js",
+  "browser": {
```

Comment:
> 
> 
> No need for platform-specific bindings

@dyladan maybe i getting u wrong, but i was thinking because if this comment i should delete browser section, am i wrong?

Reply:
No. This section exists if you need to have one behavior on web and another on node, or at least different implementations of the same behavior. For this package, the whole thing is web and node safe.